content/docs/features/bill-of-materials.md

```diff
@@ -18,76 +18,36 @@ Use the following tutorial to add a `Bill-of-Materials` using buildpacks. <br/>
 
 ## Viewing Bill of Materials
 
-You can use this command to inspect your app for it's `Bill-of-Materials`.
+You can use the `download-sbom` command to inspect your app for it's `Structured-Bill-of-Materials`. The following command will download the application layer containing the `SBoM` files to `./layers/sbom/...`.
 
 ```bash
-pack inspect-image your-image-name --bom
+pack download-sbom your-image-name
 ```
 
-It can also be accessed by looking at the label `io.buildpacks.build.metadata`. For example, running Docker CLI, jq and using the following command.
+You can also choose to download the `SBoM` from an image hosted in a remote registry, as opposed to an image hosted in a Docker daemon. You use the `--remote` flag to do so.
 
 ```bash
-docker inspect your-image-name | jq -r '.[0].Config.Labels["io.buildpacks.build.metadata"] | fromjson'
+pack download-sbom your-image-name --remote
 ```
 
-Following is the the information listed in `io.buildpacks.build.metadata` for [Sample Java App](https://github.com/buildpacks/samples/tree/main/apps/java-maven) obtained by building the app using buildpacks and running the above command.
+The following is a sample directory structure for an `SBoM` layer downloaded to the local filesystem. The files are named in the following pattern `sbom.<FORMAT>.json`, where FORMAT can be one of the Structured-Bill-of-Material formats allowed by Cloud Native Buildpacks tooling: `cdx`, `spdx`, or `syft`. Buildpack authors may choose to generate `SBoM` for the entire buildpack, or individual layers designated by the buildpack.
```

Comment:
The current phrasing is technically accurate, but it could lead to the reader thinking of lots of edge-cases.  For example, what happens if one layer contains `syft` SBoM and another layer contains `cdx`.  I think we can simplify the paragraph and focus on your syft example:

```suggestion
Cloud Native Buildpacks support `SBoM` metadata in [CycloneDX](https://cyclonedx.org/), [Syft](https://github.com/anchore/syft) or [Spdx](https://spdx.dev/) formats.  The following example demonstrates `syft` format `SBoM` metadata to the local filesystem.  The combined metadata from all of the `sbom.syft.json` files is the image `SBoM`.   Where CycloneDX `SBoM` metadata is generated, the files are named `sbom.cdx.json`.  Similarly, Spdx files are named `sbom.cdx.json`.
```